### PR TITLE
ucfunnel Bid Adapter: add support Price Floors Module

### PR DESCRIPTION
**Pull request by jackhsiehucf**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@jackhsiehucf browserstack seems to be hanging on this pr and another one I have come across and that is messing with testing. Trying to figure out what exactly the issue is.

**Comment by patmmccann** [comment]
Thanks for this PR, I removed changes to your adapter from https://github.com/prebid/Prebid.js/pull/6804

File: modules/ucfunnelBidAdapter.js [hunk 1]
```diff
@@ -210,12 +211,39 @@ function getSupplyChain(schain) {
...
+}
+
+function getFloor(bid, size, mediaTypes) {
+  if (bid.params.bidfloor) {
+    return bid.params.bidfloor;
+  }
+  if (typeof bid.getFloor === 'function') {
+    var bidFloor = bid.getFloor({
+      currency: CURRENCY,
+      mediaType: getMediaType(mediaTypes),
```
Comment (patmmccann) [inline modules/ucfunnelBidAdapter.js:230]: On multiformat unit,  this appears to slays at the video floor

Comment (jackhsiehucf) [reply modules/ucfunnelBidAdapter.js:230]: Does multiformat unit mean that adUnits support multiple mediaType ? 
Like support banner and video in the same adUnits.

Comment (jackhsiehucf) [reply modules/ucfunnelBidAdapter.js:232]: I update the commit to fix the banner bid floor problem.

Comment (jackhsiehucf) [reply modules/ucfunnelBidAdapter.js:232]: @patmmccann any comment about the update commit?

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**